### PR TITLE
Update for issue 149

### PR DESCRIPTION
**Pull request by kevin-lacoo**

### CODE REVIEW

File: src/bosh-alicloud-cpi/action/create_vm.go [hunk 1]
```diff
@@ -226,9 +226,13 @@ func (a CreateVMMethod) createVM(
 	}
 	// 为了支持CR，tag中添加创建时获取的env.bosh.group参数
 	if registryEnv.Bosh.Group != "" {
+		group_tag_value := registryEnv.Bosh.Group
```
Comment (xiaozhu36) [inline src/bosh-alicloud-cpi/action/create_vm.go:229]: group_tag_value 改成大驼峰形式： groupTagValue

File: src/bosh-alicloud-cpi/action/create_vm.go [hunk 2]
```diff
@@ -226,9 +226,13 @@ func (a CreateVMMethod) createVM(
...
+		if len(registryEnv.Bosh.Group) > 120 {
+			group_tag_value := registryEnv.Bosh.Group[:120]
```
Comment (xiaozhu36) [inline src/bosh-alicloud-cpi/action/create_vm.go:231]: 现在的限制是 128

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: src/bosh-alicloud-cpi/action/create_vm.go (hunk 1 above)
Comment (kevin-lacoo) [reply src/bosh-alicloud-cpi/action/create_vm.go]: changed, thanks

File: src/bosh-alicloud-cpi/action/create_vm.go (hunk 2 above)
Comment (kevin-lacoo) [reply src/bosh-alicloud-cpi/action/create_vm.go]: changed, thanks

**Pull request merged**